core/src/main/java/org/stellar/anchor/util/Log.java

```diff
@@ -81,7 +83,19 @@ public static void error(final String msg) {
    * @param detail The additional object to be logged.
    */
   public static void error(final String message, final Object detail) {
-    logMessageWithJson(message, detail, getLogger()::error);
+    if (detail instanceof Exception) {
+      Exception ex = (Exception) detail;
+
+      logMessageWithJson(
```

Comment:
Why is this required now? Does the message get collapsed?

Reply:
This is because `logMessageWithJson(message, detail, getLogger()::error);` failed serializing the `message` field if `detail` is an exception. 

That's the reason that we have expand it manually.